pkg/pillar/kubeapi/nitoapiserver.go

```diff
@@ -60,6 +58,24 @@ func CreateOrUpdateNAD(log *base.LogObject, nadName, jsonSpec string) error {
 	return err
 }
 
+// CheckEtherPassThroughNAD checks if network passthrough NAD is present
```

Comment:
I assume NAD means NetworkAttachmentDefinition but it wouldn't hurt spell that out in the comment.

Reply:
ok will fix that